scss/inc/_navigatorFizzy.scss

```diff
@@ -0,0 +1,312 @@
+@import 'inc/bootstrap';
+
+$lineHeight: 3rem;
+$borderWidth: 1px;
+$backgroundColor: #f2f2f2;
+$radiusCircular: 50%;
+$sizeDefault: 4rem;
+$hitboxSize: 6.25rem;
+$borderThin: 0.125rem;
+$colorTextDisabled: hsl(0, 0%, 45%);
+$colorBgDefault: white;
+$borderMedium: 0.25rem;
+$borderMediumPlus: 0.375rem;
+$colorTextDefault: hsl(0, 0%, 12%);
```

Comment:
Value seems very similar to `$textColor` from `tao-core-ui/scss/inc/_colors.scss` - can we re-use it here?

Reply:
I didn't use general colors, because this component (by design team view ) should be the same as in LDS. If i change colors a little now - we later will change it back to this colors, when starting injecting LDS into CG. I prefer to stay colors as is.

Reply:
I refactored all colors to theme variables, as you suggested